united.py

```diff
@@ -1,25 +1,51 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-
+import calendar
 import csv
 from datetime import datetime
 import os
 import argparse
+from wsgiref import headers
+from xml.etree import ElementTree
 
-from flask.ext.sqlalchemy import SQLAlchemy
+import re
+from flask_sqlalchemy import SQLAlchemy
 from sqlalchemy import and_
 
 from models import Marker, MARKER_TYPE_ACCIDENT
 from utilities import init_flask
 import importmail
 
+import urllib2
+
+from xml.dom import minidom
+from xml.dom.minidom import Document
```

Comment:
If you're already importing `minidom`, you can use `minidom.Document` rather than importing `Document` separately